### DISCUSSION TITLE
验证码测试无法重新生成

### DISCUSSION THREAD

**Discussion by eszzz** [mikumifa/biliTickerBuy#278] [提问专区]
选了本地过验证码，测试中，第一次可以生成，也可以成功过码，再点再次生成，就卡在这里生成不出来。但神奇的是，开了梯子就可以正常重新生成了，我还能正常用本地过码吗？
![屏幕截图 2024-06-30 125414](https://github.com/mikumifa/biliTickerBuy/assets/174228488/e8ff4879-880e-48ba-a496-56969b7bc483)

**Comment by mikumifa** [comment]
这个验证码加载可能要梯子？

**Reply by eszzz** [reply]
但别人说不用梯子就能加载555，那我不用管这个东西就好了嘛？

**Reply by mikumifa** [reply]
能显示一次就行了

**Reply by eszzz** [reply]
> 能显示一次就行了

好嘞，谢谢佬
